internal/services/storage/storage_account_resource_test.go

```diff
@@ -218,6 +220,9 @@ func TestAccStorageAccount_minTLSVersion(t *testing.T) {
 }
 
 func TestAccStorageAccount_allowBlobPublicAccess(t *testing.T) {
+	if features.ThreePointOhBetaResources() {
```

Comment:
can we gate this behind 3.0 and not the new resources:

```suggestion
	if features.ThreePointOh() {
```